common/lib/xmodule/xmodule/js/src/video/03_video_player.js

```diff
@@ -110,7 +110,6 @@ function (HTML5Video, Resizer) {
         state.videoPlayer.currentTime = 0;
 
         state.videoPlayer.goToStartTime = true;
-        state.videoPlayer.stopAtEndTime = true;
```

Comment:
The constant isn't used in any other context, so I thought it would be safe to remove. Could revert back to false